util/log/clog.go

```diff
@@ -836,6 +852,15 @@ func encodeLogEntry(entry *proto.LogEntry) []byte {
 	return append(data, entryData...)
 }
 
+func encodeLogEntryJSON(entry *proto.LogEntry) []byte {
+	// Marshal log entry.
+	entryData, err := json.Marshal(entry)
+	if err != nil {
+		panic(fmt.Sprintf("unable to marshal log entry: %s", err))
+	}
+	return append(entryData, '\n')
+}
+
 // processForStderr formats a log entry for output to standard error.
 func (l *loggingT) processForStderr(entry *proto.LogEntry) []byte {
 	return formatLogEntry(entry, l.shouldColorize())
```

Comment:
Yeah I can do that. I'll wait for the discussion though, would like to do some general refactoring in this code around the flags.